src/workloads/docs/Generators.yml

```diff
@@ -251,6 +251,22 @@ Actors:
             # The following example returns - repeatingSeq: [1, 2, 3, 1, 2]
             repeatingSeq: {^Array: {of: {^Choose: {from: [1, 2, 3], deterministic: true}}, number: 5}}
 
+            # You can also filter items out, similar to a set, by setting `distinct` = true.
+            # For example, you can generate N distinct random integers.
```

Comment:
In this comment you claim to generate N distinct random integers, but then you use `Repeat`, which I think undermines that? err well I guess it doesn't but this is a weird example to give folks to copy/paste. Maybe just comment that it will de-duplicate and not display that via the repeat in the docs? You can say something like:
```
Without the 'distinct' argument here we would risk generating two of the same integers.
```